opentracing-spring-web-itest/mvc/src/test/java/io/opentracing/contrib/spring/web/interceptor/itest/mvc/MVCJettyITest.java

```diff
@@ -68,129 +55,4 @@ protected TestRestTemplate getRestTemplate() {
         return testRestTemplate;
     }
 
-    @Test
-    public void testFilterException() throws Exception {
```

Comment:
This PR reduces level of visibility what is happening, it removes tracing of the standard error handling in spring boot.